stack/lab/Dockerfile

```diff
@@ -3,6 +3,13 @@ FROM base
 
 LABEL maintainer="AiiDAlab Team <aiidalab@materialscloud.org>"
 
+# The lab mode is default to run jupyter server.
+# Here the CMD pinned to `notebook` for running in legacy notebook backend
+# In the future, we have appmode compatible with nbclassic
```

Comment:
Do I understand correctly that Appmode is currently not compatible with the `nbclassic` option?